notifier/orb.yaml

```diff
@@ -1,12 +1,56 @@
 version: 2.1
 description: "Tools for notifying external services about CI events"
 
+orbs:
+  slack: circleci/slack@4.7.0
```

Comment:
Turns out `4.7.1` is already available. In general it's good to avoid overspecifying dependency versions in libraries (ex. orbs). You can use `circleci/slack@4` to maintain forward compatibility while keeping this orb dependency flexible.

Reply:
Oh! This feature is great. I will replace `@4.7.0` for `@4`.